doc/conf.py

```diff
@@ -63,9 +63,6 @@
 # The encoding of source files.
 # source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
```

Comment:
I'd expect this [to be a problem](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-root_doc):

> Changed in version 2.0: The default is changed to 'index' from 'contents'.

And in 4.0 (not out yet) it changes to `root_doc`.

But maybe it won't care if there is no `contents.rst`, or maybe it will fall back to `index.rst` in that case? TBD

Reply:
Good point, I forgot about the sphinx<2.0 support. I'll put the references to `master_doc` back in with a comment about the upcoming name change in sphinx 4.

Reply:
This isn't needed.  The default value for both `master_doc` and `root_doc` is `index`.  If you delete this, then everything works.